src/main/java/application/rest/v1/CustomResourceWatcher.java

```diff
@@ -137,9 +148,56 @@ private static void setLoggerLevel(JsonObject o) {
                             if (level.equals("all"))
                                 Logger.setLogLevel(Logger.LogLevel.ALL);    
                         }
+                        
+                        //check for ui setting
+                        JsonElement levelUi  = loggingObj.get("ui"); 
+                        if (UI_LOG_LEVEL_API != null && levelUi != null && levelUi.isJsonPrimitive())  {               
+                            String level = levelUi.getAsString();
+                            
+                            if(!CURRENT_UI_LOG_LEVEL.equals(level)) {
+                                setUILogLevel(level); 
+                            }
+                        }
                     }
                 } 
             }
         }                                                    
     }
- }
\ No newline at end of file
+
+    public static void setUILogLevel(String level) {
+        String methodName = "setUILogLevel";
+        if (Logger.isEntryEnabled()) {
+            Logger.log(CLASS_NAME, methodName, Logger.LogType.ENTRY, level);
+        }
+
+        //match levels with levels that the UI supports
+        String newLevel = level;
+        if (level.equals("warning")) {
```

Comment:
Does this code map all of the possible log level values (e.g. info, all) on to the levels that the UI supports?

Reply:
Yes, the other levels match what the documentation says.